stylus/components/button.styl

```diff
@@ -207,8 +206,10 @@ $button-alert--info
 
     &:active
     &:hover
+    &:not([disabled]):not([aria-disabled=true]):hover
```

Comment:
This is not cool, but there's another selector like that earlier in the file that forces a blue background on all buttons. Any recommendation @GoOz ?